core/metadata-3.2.md

```diff
@@ -201,6 +253,8 @@ following labels and formats:
 | 767 | `ERR_KEYINVALID`      | `<Key> :invalid metadata key`            |
 | 768 | `ERR_KEYNOTSET`       | `<Target> <Key> :key not set`            |
 | 769 | `ERR_KEYNOPERMISSION` | `<Target> <Key> :permission denied`      |
+| 770 | `ERR_METADATARATELIMIT` | `<Target> <Key> [<RetryAfter>]`        |
+| 771 | `ERR_METADATASYNCLATER` | `<Target> [<RetryAfter>]`               |
 
```

Comment:
These clash with the numerics in #250

The test implementation in inspircrd uses:

`ERR_METADATASYNCLATER = 774`

https://github.com/inspircd/inspircd/commit/3d52a33d3c650292b33825ce3458456790149caa#diff-410482bb2c797687c7731b0efe472abeR38

We should also set

`ERR_METADATARATELIMIT = 775`